xml/x509_subjects.xml

```diff
@@ -1688,11 +1688,14 @@
   <fingerprint pattern="^CN=iSTAR Ultra">
     <description>iSTAR Ultra</description>
     <example>CN=iSTAR Ultra,OU=Access Control and Video Division,O=Johnson Controls,L=Westford,ST=Massachusetts,C=US</example>
-    <param pos="0" name="os.vendor" value="iStar"/>
-    <param pos="0" name="os.product" value="Linux"/>
-    <param pos="0" name="hw.vendor" value="iStar"/>
+    <param pos="0" name="os.vendor" value="Software House"/>
```

Comment:
@hdm @pbarry25 Since the iSTAR Ultra fingerprint was recently added (#382) I wanted to check with you to see if you have any concerns with these changes.